src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -257,6 +257,28 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         return vs;
     }
 
+    // This is used for defterm connections. The app will get these from the
```

Comment:
NAK on a generic named method on a public interface being _for a specific use case only._ Either the name should be suitably nongeneric or the method's behavior suitable widened!